ssrando/ssrando
# Shuffle batreaux crystal counts

## Additions
New option: Batreaux Crystal Counts:
- Vanilla: the normal counts (5, 10, 30, 40, 50, 70, 80)
- Half: half of the normal counts (2, 5, 15, 20, 25, 35, 40)
- Random: completely random counts (min 1, max 80)

Batreaux will become human when the last reward is redeemed, regardless of how many crystals he requires
Kukiel will now always appear in Batreaux's house and tells you how many crystals Batreaux requires
Batreaux also tells you how many crystals he wants for his next reward if you speak to him without enough crystals, as he would in vanilla

## How to test
I'd recommend playing with the random option, it's the most fun and best way to test this feature. Check with Kukiel in Batreaux's house to see how many crystals he requires, then redeem his rewards as you get enough crystals throughout your playthrough.

## Notes
Batreaux's chest will still appear along with his third reward (vanilla is 30 crystals)
Batreaux will give 2 rewards for the second-to-last crystal count (as he gives 2 rewards for 70 crystals in vanilla)
Batreaux's checks are all renamed to something along the lines of `Batreaux's House - First Reward`.
Crystal counts will appear in the spoiler log
